examples/tv-app/android/App/.idea/gradle.xml

```diff
@@ -14,7 +14,6 @@
             <option value="$PROJECT_DIR$/platform-app" />
           </set>
         </option>
-        <option name="resolveModulePerSourceSet" value="false" />
```

Comment:
What does this do?

Reply:
It was auto added by android studio when opening the file